### PR TITLE
alllow for default language jft sms if available #147

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

**Comment by pjaudiomv** [comment]
I can change this back to use Sms verb as well if you think i should

**Comment by radius314** [comment]
Have you tested this with ngrok?

**Comment by pjaudiomv** [comment]
yea I gotta get used to using ngrok, i just tested it with a test twilio number of mine. Ill add french canadian and test again.

**Comment by radius314** [comment]
If you want we can do a session again with how to set up a test environment locally.

**Comment by pjaudiomv** [comment]
yea that would be helpful, i gotta jot basic instructions down in a txt file just for a refresh. I have a txt file for git commands i don't use much

**Comment by pjaudiomv** [comment]
damn it that function doesn't even have to be there anymore, as im not using it. however 1600 is the character limit for concatenated messages with twilio, meaning they wont auto concatenate a message with a body longer than 1600 chrs

File: sms-gateway.php [hunk 1]
```diff
@@ -19,13 +26,12 @@
...
     } 
 	else if (str_exists(strtoupper($address), strtoupper('jft'))) {
-         $result = get("https://www.jftna.org/jft/");
-         $stripped_results = strip_tags($result);
-         $without_tabs = str_replace("\t", "", $stripped_results);
-	     $without_htmlentities = html_entity_decode($without_tabs);
-	     $without_extranewlines = preg_replace("/[\r\n]+/", "\n\n", $without_htmlentities); ?>
-         <Sms><?php  echo $without_extranewlines; ?> </Sms>
- <?php } 
+         $jft = get_jft(true);
```
Comment (radius314) [inline sms-gateway.php]: Can we stick with TwiML?  Just use `<Message>`

**Comment by radius314** [comment]
#147

**Comment by pjaudiomv** [comment]
https://support.twilio.com/hc/en-us/articles/223181028-Switching-from-SMS-Messages-resource-URI-to-Messages-resource-URI-

https://support.twilio.com/hc/en-us/articles/223181508-Does-Twilio-support-concatenated-SMS-messages-or-messages-over-160-characters-

**Comment by radius314** [comment]
cool thanks for the articles

**Pull request merged**